src/components/ViewRequest/Actions/ReworkRequest.tsx

```diff
@@ -7,28 +7,49 @@ import {
   DialogSurface,
   DialogTitle,
   DialogTrigger,
+  Field,
+  Textarea,
+  TextareaProps,
   Tooltip,
 } from "@fluentui/react-components";
 import { NavigateBackIcon } from "@fluentui/react-icons-mdl2";
+import { useAddNote } from "api/notesApi";
 import { useRequest, useUpdateStage } from "api/requestsApi";
 import { STAGES } from "consts/Stages";
+import { useState } from "react";
 import { useParams } from "react-router-dom";
 
+declare const _spPageContextInfo: {
+  userId: number;
+  userDisplayName: string;
+  userEmail: string;
+  userLoginName: string;
```

Comment:
Can possibly reduce this to just have the one prop that we use in this component